test/test_nn.py

```diff
@@ -8281,6 +8281,12 @@ def _gelu_ref(X):
                 _test_gelu(n, m, torch.float64, True)
                 _test_gelu(n, m, torch.float64, False)
 
+        # Tesst multi threaded
+        num_threads = torch.get_num_threads()
+        torch.set_num_threads(4)
+        _test_gelu(32, 32, torch.float32, False)
+        torch.set_num_threads(num_threads)
```

Comment:
cc @mruberry

Reply:
Thanks for the fyi. Creating a context manager for this would be nicer. Note that if the test fails the number of threads won't be reset properly here. Although this PR doesn't need to add a context manager, the reset to the original `num_threads` should happen in a `finally` block:

```
num_threads = torch.get_num_threads()
torch.set_num_threads(4)
try:
  _test_gelu(32, 32, torch.float32, False)
finally:
  torch.set_num_threads(num_threads)
```

Reply:
Thats a good point. Thanks for the suggestion. Will fix.